io_chan.go

```diff
@@ -1,20 +1,29 @@
 package main
 
-import "io"
+import (
+	"io"
+	"time"
+)
+
+// Simulates a TCP packet by storing a slice of bytes with the receive timestmap
```

Comment:
TCP packet doesn't really make sense, rather you're buffering a stream—but I suppose that's what you mean?